app/cmd/preview.go

```diff
@@ -142,6 +187,10 @@ func previewCmdError(msg string) {
 	os.Exit(1)
 }
 
+func printlnGreen(text string) {
+	fmt.Printf("\033[32m%s\033[0m\n", text)
```

Comment:
omg now I want all the colors

Reply:
same I thought about making a switch helper that you could get like 8 colors from instead of just green... We don't need it.. But maybe we do!!!!!!!!!!!!!!!!!